components/Carousel/sub-components/CarouselCard.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import ThumbCard from './ThumbCard';
+import DescriptionCard from './DescriptionCard';
+import { colors, spacing, baseFontSize } from '../../shared/theme';
+
+const renderCarouselCardSize = (card, cardSize) => {
+  if (cardSize === 'small') return <ThumbCard card={card} />;
+
+  return <DescriptionCard card={card} cardSize={cardSize} />;
+};
+
+const CarouselCard = ({ card, cardSize, theme }) => (
+  <>{renderCarouselCardSize(card, cardSize, theme)}</>
```

Comment:
this is set `theme` but does not using in `renderCarouselCardSize`. Is that right?